### PR TITLE
Updated python modules

### PR DESCRIPTION
**Pull request by truthfool**
- Updated requirements for python modules
- black formatting
- Added environment variables for secret and urls
@avikganguly01 @fynmanoj

### CODE REVIEW

File: openg2p_disbursement/models/openg2p_disbursement_batch_transaction.py [hunk 1]
```diff
@@ -226,20 +226,20 @@ def create_bulk_transfer(self):
...
+            "data": open(csvname, "rb"),
         }
         headers = {
-            'Purpose': 'test payment',
-            'filename': csvname,
-            'X-CorrelationID': 'bd85c0e3-b7bd-40aa-b00f-3240df9d69bd',
-            'Platform-TenantId': os.environ.get("tenantName"),
+            "Purpose": "test payment",
+            "filename": csvname,
+            "X-CorrelationID": "bd85c0e3-b7bd-40aa-b00f-3240df9d69bd",
```
Comment (fynmanoj) [inline openg2p_disbursement/models/openg2p_disbursement_batch_transaction.py:237]: Correlation id should be generated for each call, this should not be hardocded.

File: openg2p_disbursement/models/openg2p_disbursement_batch_transaction.py [hunk 2]
```diff
@@ -251,16 +251,21 @@ def create_bulk_transfer(self):
     def get_auth_token(self):
         try:
             headers = {
-                'Platform-TenantId': os.environ.get("tenantName"),
-                'Authorization': 'Basic Y2xpZW50Og==',
-                'Content-Type': 'text/plain',
+                "Platform-TenantId": os.environ.get("tenantName"),
+                "Authorization": "Basic Y2xpZW50Og==",
```
Comment (fynmanoj) [inline openg2p_disbursement/models/openg2p_disbursement_batch_transaction.py:255]: this field should not be added to code, as it is a credential, it should be configured outside code

File: openg2p_disbursement/models/openg2p_disbursement_batch_transaction.py [hunk 3]
```diff
@@ -251,16 +251,21 @@ def create_bulk_transfer(self):
...
-                'username': os.environ.get("username"),
-                'password': os.environ.get("password"),
-                'grant_type': os.environ.get("grant_type"),
+                "username": os.environ.get("username"),
+                "password": os.environ.get("password"),
+                "grant_type": os.environ.get("grant_type"),
             }
-            response = requests.post('http://ops-bk.sandbox.fynarfin.io/oauth/token', params=params, headers=headers,verify=False)
+            response = requests.post(
+                "http://ops-bk.sandbox.fynarfin.io/oauth/token",
```
Comment (fynmanoj) [inline openg2p_disbursement/models/openg2p_disbursement_batch_transaction.py:264]: this URL should not be added to the code, this should be removed from the code and, should be added as a config or environment variable :  eg: `os.environ.get("ops-url"),`

File: openg2p_disbursement/models/openg2p_disbursement_batch_transaction.py [hunk 4]
```diff
@@ -226,20 +226,20 @@ def create_bulk_transfer(self):
         url = "http://bulk-connector.sandbox.fynarfin.io/batchtransactions"
```
Comment (fynmanoj) [inline openg2p_disbursement/models/openg2p_disbursement_batch_transaction.py:226]: please remove this URL from the code and make it configurable or env variable

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: openg2p_disbursement/models/openg2p_disbursement_batch_transaction.py (hunk 1 above)
Comment (truthfool) [reply openg2p_disbursement/models/openg2p_disbursement_batch_transaction.py:237]: Okay I will pass this as a new UUID each time.

File: openg2p_disbursement/models/openg2p_disbursement_batch_transaction.py (hunk 2 above)
Comment (truthfool) [reply openg2p_disbursement/models/openg2p_disbursement_batch_transaction.py:255]: Its taking from environment variable .

File: openg2p_disbursement/models/openg2p_disbursement_batch_transaction.py (hunk 3 above)
Comment (truthfool) [reply openg2p_disbursement/models/openg2p_disbursement_batch_transaction.py:264]: Will make it to add it from environment variable.

File: openg2p_disbursement/models/openg2p_disbursement_batch_transaction.py (hunk 4 above)
Comment (truthfool) [reply openg2p_disbursement/models/openg2p_disbursement_batch_transaction.py:226]: Will add it to environment variable

File: openg2p_disbursement/models/openg2p_disbursement_batch_transaction.py (hunk 2 above)
Comment (fynmanoj) [reply openg2p_disbursement/models/openg2p_disbursement_batch_transaction.py:256]: ` "Authorization": "Basic Y2xpZW50Og==",`   This line

**Review by fynmanoj — APPROVED** [review]

**Pull request merged**